assembl/static2/js/app/components/administration/exportSection.jsx

```diff
@@ -0,0 +1,161 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n, Translate } from 'react-redux-i18n';
+import { Link } from 'react-router';
+import { compose, graphql } from 'react-apollo';
+import { FormGroup, Radio, FormControl } from 'react-bootstrap';
+import withLoadingIndicator from '../../components/common/withLoadingIndicator';
+
+import { getDiscussionId } from '../../utils/globalFunctions';
+import SectionTitle from './sectionTitle';
+import DiscussionPreferenceLanguage from '../../graphql/DiscussionPreferenceLanguage.graphql';
+
+type Props = {
+  languages?: Array<Object>,
+  exportType: string,
+  voteSessionId?: string
+};
+
+type State = {
+  exportLocale: ?string,
+  translate: boolean
+};
+
+class DumbExportSection extends React.Component<Object, Props, State> {
+  props: Props;
+
+  state: State;
+
+  static defaultProps = {
+    languages: null,
+    voteSessionId: null
+  };
+
+  static ExportLanguageDropDown = ({ languages, onSelect, activeKey }) => {
```

Comment:
I thought there is no language select on this page but maybe I misunderstood (maybe check with Fred). If it stays, I would put this in a component in `common/` and use it in both export sections. I guess you should put all the translation mechanism in a component also (the radio buttons and state management).